src/index.ts

```diff
@@ -242,27 +214,29 @@ async function wranglerCommands() {
 			commands.push(deployCommand);
 		}
 
-		const arrPromises = commands.map(async (command) => {
-			if (environment.length > 0 && !command.includes(`--env`)) {
-				command = command.concat(` --env ${environment}`);
-			}
+		for (let command of commands) {
+			const args = ["wrangler", ...command.split(/\s+/)];
```

Comment:
This is hoping the command has no quotes strings with spaces

We might want to document this to also allow `command` to be an array as an escape hatch

Reply:
I think `@actions/exec` handles that scenario for us with its escaping logic, but worth looking into to confirm that.

Reply:
you were right, great catch. I've removed the `split` here and instead now pass the provided command (prepended with `npx wrangler `) as part of the first argument to `exec()`, which is expected to be a string that has already been escaped. The second array argument is now used to provide any additional options we want to inject into the users command (e.g. `--env ...` or `--vars ...`), which `exec()` will escape for us.